sqlite.go

```diff
@@ -93,6 +111,8 @@ var Open sqliteh.OpenFunc = func(string, sqliteh.OpenFlags, string) (sqliteh.DB,
 // code between calls to rows.Next.
 type TraceFunc func(prepCtx context.Context, query string, duration time.Duration, err error)
 
+type ConnInitFunc func(ctx context.Context, conn driver.ConnPrepareContext) error
```

Comment:
Document what state conn is in when a ConnInitFunc gets called. What things can you / can you not do in this function? What happens if you return an error?